response_operations_ui/templates/collection-exercise.html

```diff
@@ -8,8 +8,21 @@
     <h1 class="jupiter">{{survey.shortName}} {{ce.name}}</h1>
 
     <div class="grid">
-      <div class="grid__col col-5@l"></div>
-      <div class="grid__col col-1@l"></div>
+      <div class="grid__col col-6@l">
+        <section class="ce-section">
+          <h2 class="venus u-pt-m">Collection instruments</h2>
+          <div class="u-mb-m">
+            <form action="" class="form">
+              <p>
+                Choose a collection instrument (CI) to load
+              </p>
+              <label class="upload-file u-vh" for="file">Choose CI file</label>
```

Comment:
I don't think we have the `u-vh` class in the css. 
I added it into the `base.scss` for my PR